spec/features/process_a_request_accept_spec.rb

```diff
@@ -25,6 +25,7 @@
 
     before do
       switch_feature_flag_with(:staff_prisons_with_nomis_contact_list, [prison.name])
+      switch_feature_flag_with(:staff_prisons_with_slot_availability, [])
```

Comment:
Turned out I'd enabled prison slot availability for that prison which got picked up by the test env and caused this test to fail, so I think it is needed to keep the test ENV change proof